crates/rattler_package_streaming/src/seek.rs

```diff
@@ -84,6 +85,25 @@ fn get_file_from_archive(
     Err(ExtractError::MissingComponent)
 }
 
+/// Read a package file content from archive
+fn read_package_file_content<'a, P: PackageFile>(
```

Comment:
Maybe we can just pass the `package_path` in, as afaics it does not use the generic for anything else?

Reply:
`file` is used by `stream_tar_bz2` to read form file, and `path` is used to determine `ArchiveType` so we need both

Reply:
Sorry I dont get what you mean? AFAICS the only place that generic is sued in this function is for the `P::package_path`? And if you just pass that in as a str it makes the method more generic by default.